content/id/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,188 @@
+Otomasi pada umumnya dapat bekerja dengan berbagai macam kluster Kubernetes, termasuk
+kluster yang terhosting dan instalasi yang diatur.
+
+Ada pola spesifik tertentu untuk menulis program klien yang bekerja baik dengan Kubernetes
+yang disebut pola *Controller*. *Controller* biasanya membaca objek milik `.spec`, kemungkinan
+melakukan sesuatu, dan kemudian memperbarui objek milik `.status`.
+
+*Controller* adalah klien dari Kubernetes. Ketika Kubernetes adalah klien dan memanggil layanan
+jarak jauh, ini disebut *Webhook*. Layanan jarak jauhnya disebut *Webhook Backend*. Seperti *controller*, Webhooks
+memang menambah poin kegagalan.
+
+Di dalam model *Webhook*, Kubernetes membuat sebuah *network request* kepada sebuah layanan jarak jauh.
+
+Di dalam model *Binary Plugin*, Kubernetes mengeksekusi file binari (program).
+Plugin binari digunakan oleh kubelet (seperti [Plugin Volume Flex](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md)
+dan [Plugin Jaringan](/docs/concepts/cluster-administration/network-plugins/)) dan oleh kubectl.
+
+Berikut ini adalah diagram yang menunjukkan bagaimana titik ekstensi berinteraksi dengan bidang kontrol Kubernetes.
+
+<img src="https://docs.google.com/drawings/d/e/2PACX-1vQBRWyXLVUlQPlp7BvxvV9S1mxyXSM6rAc_cbLANvKlu6kCCf-kGTporTMIeG5GZtUdxXz1xowN7RmL/pub?w=960&h=720">
+
+<!-- image source drawing https://docs.google.com/drawings/d/1muJ7Oxuj_7Gtv7HV9-2zJbOnkQJnjxq-v1ym_kZfB-4/edit?ts=5a01e054 -->
+
+
+## Titik-titik Ekstensi
+
+Diagram berikut menunjukkan titik-titik ekstensi di sistem Kubernetes.
+
+<img src="https://docs.google.com/drawings/d/e/2PACX-1vSH5ZWUO2jH9f34YHenhnCd14baEb4vT-pzfxeFC7NzdNqRDgdz4DDAVqArtH4onOGqh0bhwMX0zGBb/pub?w=425&h=809">
+
+<!-- image source diagrams: https://docs.google.com/drawings/d/1k2YdJgNTtNfW7_A8moIIkij-DmVgEhNrn3y2OODwqQQ/view -->
+
+1. Pengguna biasa berinteraksi dengan API Kubernetes menggunakan `kubectl`. [Kubectl plugins](/docs/tasks/extend-kubectl/kubectl-plugins/) memperluas binari kubectl. Mereka hanya memengaruhi lingkungan lokal pengguna, dan tidak dapat menegakkan kebijakan di seluruh situs.
+2. API server menangani semua permintaan. Beberapa tipe titik ekstensi di API server memperbolehkan otentikasi permintaan, atau memblokir mereka berdasarkan konten, konten editing, dan penanganan penghapusan mereka. Hal ini dideskripsikan di bagian [Ekstensi Akses API](/docs/concepts/overview/extending#api-access-extensions)
+3. API server melayani berbagai macam *resources*, *Built-in resource kinds*, seperti `pods`, didefinisikan oleh projek Kubernetes dan tidak dapat diubah. Anda juga dapat menambahkan sumber yang Anda definisikan sendiri, atau yang projek lain definisikan, memanggil *Custom Resources*, seperti yang dijelaskan di bagian [Sumber Daya Kustom](/docs/concepts/overview/extending#user-defined-types). Sumber khusus sering digunakan dengan Ekstensi API Aksi.
```

Comment:
```suggestion
3. API server melayani berbagai macam sumber daya. _Tipe-tipe sumber daya bawaan_, seperti `Pod`, didefinisikan oleh proyek Kubernetes dan tidak dapat diubah. Anda juga dapat menambahkan sumber daya yang Anda definisikan sendiri, atau yang proyek lain definisikan, disebut _Custom Resources_, seperti dijelaskan di bagian [Sumber Daya _Custom_](/docs/concepts/overview/extending#user-defined-types). Sumber daya _Custom_ sering digunakan dengan Ekstensi Akses API.
```

Kustom is not an Indonesian word. Let's fallback to the English word to avoid confusion

Reply:
done